Update items.md

SirLich opened the pull request:
Adds the animation and sound component for food items.
- added it in the forst example on the top of the page
- added the two component examples for eating food or drinking a drink (which add both the animations and the sounds)

pieterdefour committed:
Update items.md

Adds the animation and sound component for food items.
- added it in the forst example on the top of the page
- added the two component examples for eating food or drinking a drink (which add both the animations and the sounds)

pieterdefour commented:
I think I need to change it so it doenst have two times the code, but instead just the one with "eat" and on the side of that a //Adds the animation and sounds when eating a food item. Can also be changed to "drink".

SirLich commented:
@pieterdefour You can make a new update/commit, and it will automatically go here

pieterdefour committed:
Update items.md

Changed the placement of the minecraft:use_animation component to below the minecraft:food component.
Changed the example to only show the "eat" example with some extra info next to it for the "drink" example.
